src/targets/index.ts

```diff
@@ -65,6 +66,8 @@ export function handlebarsInstance(tmplPath: string, partialsDir: string): Templ
     return content.split("\n").map(s => `${padding}${s}`.trimRight()).join("\n")
   })
 
+  instance.registerHelper("upperFirst", _.upperFirst)
```

Comment:
Processing of all syntactical data must occur before being given to the templates, so this additional helper is indicative of a code smell.